<!-- default launch -->
<launch>

  <!-- Arguments -->
  <arg name="node_name"               default="TREK1000"/>
  <arg name="tagDistance_raw"         default="/$(arg node_name)/tagDistance_raw"/>
  <arg name="tagDistance_corrected"   default="/$(arg node_name)/tagDistance_corrected"/>
  <arg name="anchorDistance"          default="/$(arg node_name)/anchorDistance"/>
  <arg name="serial_port"             default="/dev/ttyACM0" />
  <arg name="config_file"             default="$(find evb1000_driver)/launch/default_config.yaml" />

  <node name="$(arg node_name)" pkg="evb1000_driver" type="TREK1000_node"  output="screen">
    <param name="device"                  value="$(arg serial_port)"/>
    <remap from="/tagDistance_raw"        to="$(arg tagDistance_raw)"/>
    <remap from="/tagDistance_corrected"  to="$(arg tagDistance_corrected)"/>
    <remap from="/anchorDistance"         to="$(arg anchorDistance)"/>
    <rosparam file="$(arg config_file)"   command="load" />
  </node>
</launch>
